rbf-chunker/src/main.rs

```diff
@@ -88,15 +85,17 @@ impl SectionInfo {
     }
 }
 
-const PACKET_MIN_SIZE: usize = 916;
-const PACKET_MAX_SIZE: usize = 916;
-const DISCARD_MAX_SIZE: usize = PACKET_MAX_SIZE;
+const PACKET_MIN_SIZE: usize = 524;
+const PACKET_MAX_SIZE: usize = 524;
```

Comment:
Given the Cyclone V GX stuff is 916 bytes, this seems strange.

Reply:
Yeah, at first I thought it was a bug in the chunker, but then I counted the number of zeros in those all-zero packets and got 522. Then added 2 (for the crc), and huzzah! It worked.